leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -160,8 +161,8 @@ protected void track(TrackEvents event) {
   private void setUpMainVariables() {
     Geocoder geocoder = new Geocoder(this, Locale.getDefault());
     apiInteractor = new GeocoderAPIInteractor(new NetworkClient(), new AddressBuilder());
-    geocoderPresenter = new GeocoderPresenter(new ReactiveLocationProvider(getApplicationContext()),
-        new GeocoderInteractor(geocoder), apiInteractor);
+    GeocoderRepository geocoderRepository = new GeocoderRepository(new GeocoderInteractor(geocoder), apiInteractor);
```

Comment:
Looks a bit messy. A repository using interactors?
Shouldn't it be Activity -> presenter -> interactor -> repository -> datasource?